<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagesManagerWindow</class>
 <widget class="QWidget" name="ImagesManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление изображениями</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="moveAbove">
     <property name="text">
      <string>↑</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="moveBelow">
     <property name="text">
      <string>↓</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListView" name="imagesView"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="addImage">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="removeImage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
